.github/workflows/ci-windows.yml

```diff
@@ -0,0 +1,32 @@
+name: CI (windows)
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  run-ci:
+    name: Run
+
+    runs-on: windows-2019
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup NuGet
```

Comment:
The tests passed even without this set up. Do you think it should be kept for "good practice" or should I remove it? @walro

Reply:
If the action is not needed, we should remove it :)